powershell/include.ps1

```diff
@@ -1,6 +1,8 @@
-. $PSScriptRoot\spl-variables.ps1
-. $PSScriptRoot\spl-functions.ps1
+# Always set the $InformationPreference variable to "Continue"

```

Comment:
The comment seems useless as it is set to Continue here. A comment what this means would be nice.